driver/razeranansicommon.c

```diff
@@ -0,0 +1,14 @@
+#include "razeranansicommon.h"
+
+struct razer_report razer_anansi_enable_macro_flash()
+{
```

Comment:
This command technically isnt specific to the anansi... it just isnt used elsewhere. (its listed in the fw spec I have, it just listed without any description ha)

Replace with the following and put that in razerchromacommon.c, and the appropriate entry into razerchromacommon.h, then we have 1 less file.
```c
struct razer_report razer_chroma_standard_set_led_blinking_params(unsigned char variable_storage, unsigned char led_id)
{
	struct razer_report report = get_razer_report(0x03, 0x04, 0x04);
	report.arguments[0] = variable_storage; // Effect ID
	report.arguments[1] = led_id; // LED ID
	report.arguments[2] = 0x05;
	report.arguments[3] = 0x05;
	
	return report;
}
```


Where variable_storage would be one of the defines NOSTORE or VARSTORE and led_id should be MACRO_LED,

I have a feeling arg 2,3 could possibly be on and off timing.